modules/ROOT/pages/introduction/intro-config-use-studio.adoc

```diff
@@ -0,0 +1,159 @@
+= Use Studio to Configure a Connector
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+To use Anypoint Studio to configure a connector:
+
+. Install the connector.
+. Configure an input source for the connector.
+. Configure the connector.
+
+Follow these best practices when you configure an input source and connector in Studio:
+
+*  Configure an associated global element for the input source and connector.
+* Create a `.yaml` file or `.properties` file to contain your properties, and then reference the file from the configuration.
+* Configure a reconnection strategy for the input source and target connector.
+
+For information about these best practices, see <<introduction/intro-connector-configuration-overview.adoc#best-practices,Best Practices for Configuring Connectors>>.
+
+== Install the Connector
+
+Installing the connector adds it to a Studio project. Studio provides two ways to install a connector:
+
+* Use the Exchange application in the Studio taskbar.
+
+* Use the Exchange icon in the Mule Palette view.
+
+(*Reviewers, is there any advantage to using one of these methods for installing a connector* )
+
+=== Install the Connector from the Studio Taskbar
+
+. In Studio, create a Mule project.
+. Click the Exchange icon *(X)* in the upper left of the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
```

Comment:
The credentials settings should be outside these steps. When users add a new connector from the Mule Pallete view or Exchange, in both cases they'll need to provide credentials. 

I would move the Anypoint Platform configuration step as a prerequesite under **To use Anypoint Studio to configure a connector:**
You can link to this task in the Studio docs: https://docs.mulesoft.com/studio/7.4/set-credentials-in-studio-to

Also, this topic would only work for US cloud users. EU Cloud, and PCE users need to specifically configure Studio to point to their organizations: https://docs.mulesoft.com/studio/7.4/pce-configuration